spezi-module/contact/src/androidTest/java/edu/stanford/spezikt/spezi_module/contact/ContactScreenTest.kt

```diff
@@ -0,0 +1,109 @@
+package edu.stanford.spezikt.spezi_module.contact
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Call
+import androidx.compose.material.icons.filled.Email
+import androidx.compose.material.icons.filled.Info
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import edu.stanford.spezikt.spezi_module.contact.model.Contact
+import edu.stanford.spezikt.spezi_module.contact.model.ContactOption
+import edu.stanford.spezikt.spezi_module.contact.model.ContactOptionType
+import edu.stanford.spezikt.spezi_module.contact.repository.ContactRepository
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.whenever
+import java.util.UUID
+
+class ContactScreenTest {
+
+    private val mockContactRepository: ContactRepository = mock()
```

Comment:
No need for mocking repository at this stage since it is not used in the scree at all. You can replace this with `private val contact = createContact()`. I am sure that `mockito` has their own tests that verify that their stubbing works as expected 😄